third_party/2and3/flask/testing.pyi

```diff
@@ -4,17 +4,22 @@
 
 from click import BaseCommand
 from click.testing import CliRunner, Result
-from typing import Any, IO, Iterable, Mapping, Optional, Union
+from typing import Any, IO, Iterable, Mapping, Optional, Union, Tuple, overload
+from typing_extensions import Literal
 from werkzeug.test import Client
+from flask.wrappers import Response
 
 def make_test_environ_builder(app: Any, path: str = ..., base_url: Optional[Any] = ..., subdomain: Optional[Any] = ..., url_scheme: Optional[Any] = ..., *args: Any, **kwargs: Any): ...
 
-class FlaskClient(Client):
+class FlaskClient(Client[Response]):
```

Comment:
Will this really always use a `Response`? As far as I can see, `FlaskClient` does not initialize `response_wrapper`.

Reply:
You're right: it's the `app.test_client` method that initialize it. Done.